webapp/app/forms/fields.py

```diff
@@ -25,9 +25,18 @@ def _add_classes_to_kwargs(kwargs, classes):
 
 class NumericInputMixin:
 
-    @staticmethod
-    def set_inputmode(kwargs):
+    @classmethod
+    def set_inputmode(cls, kwargs):
         kwargs.setdefault('inputmode', 'numeric')
+
+        return kwargs
+
+
+class NumericInputMaskMixin(NumericInputMixin):
```

Comment:
Love the explicit name, maybe the other one should be called `NumericInputmodeMixin` to be similarly explicit?

Also, I don't think this should inherit from the other mixin. Having `SteuerlotseIntegerField` etc inherit from both mixins would make it clearer to me what's going on.

Reply:
Good idea. We can then also use a static method above.

Reply:
I had a look at it and I am not longer sure if this is a good path to take.

We use `set_inputmode` e.g in SteuerlotseNumericStringField in `__call__` in the following way:
```python
    def __call__(self, *args, **kwargs):
        kwargs = self.set_inputmode(kwargs)
        #...
```

However, if we use multiple inheritance this will only look for the first set_inputmode in the inheritance tree and this would be NumericInputmodeMixin for `SteuerlotseNumericStringField(NumericInputModeMixin, NumericInputMaskMixin, SteuerlotseStringField)`. So either we keep it like this or we change the method names to set_inputmode and set_masking.

I think that I like making it more explicit with extra method names more